TextSelection/Views/MultitouchTextView.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  MultitouchTextView.swift
+//  TextSelection
+//
+//  Created by Lacy Rhoades on 5/15/16.
+//  Copyright © 2016 Ink & Switch. All rights reserved.
+//
+
+import UIKit
+
+class MultitouchTextView: UITextView {
+    private var highlightSelection: Selection?
+    private var preSelection: Selection?
+    private var selection: Selection?
+    
+    var customDelegate: MultitouchTextViewDelegate?
+    
+    override init(frame: CGRect, textContainer: NSTextContainer?) {
+        super.init(frame: frame, textContainer: textContainer)
+        
+        self.setup()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        
+        self.setup()
+    }
+    
+    func setup() {
+        self.editable = false
+        
+        self.gestureRecognizers?.forEach({ (recognizer) in
+            switch Mirror(reflecting: recognizer).subjectType {
+            case is UILongPressGestureRecognizer.Type:
+                // For normal selections
+                self.removeGestureRecognizer(recognizer)
+                break
+            case is UITapGestureRecognizer.Type:
+                // Not sure what this is
+                // Probably "cancels" selections
+                self.removeGestureRecognizer(recognizer)
+                break
+            case is UIPanGestureRecognizer.Type:
+                // For scrolling
+                let pan = recognizer as! UIPanGestureRecognizer
+                pan.minimumNumberOfTouches = 2
+                break
+            default:
+                let className = String(Mirror(reflecting: recognizer).subjectType)
+                if (className == "UITapAndAHalfRecognizer") {
+                    // For tap and drag
+                    self.removeGestureRecognizer(recognizer)
```

Comment:
I have to find this one by it's String-ified class name because the actual class is hidden inside the Apple libraries.